### PR TITLE
First step into nicer Channel menu, when in poster mode

### PR DESCRIPTION
**Pull request by ukdtom**
modified:   Contents/Code/interface/menu.py
	modified:   Contents/Code/interface/menu_helpers.py
	modified:   Contents/Code/support/items.py

### CODE REVIEW

File: Contents/Code/interface/menu.py [hunk 1]
```diff
@@ -24,6 +23,8 @@
 route = enable_channel_wrapper(route)
 # noinspection PyUnboundLocalVariable
 handler = enable_channel_wrapper(handler)
+# default thumb
+thumb=R(ICON)
```
Comment (pannal) [inline Contents/Code/interface/menu.py]: Doesn't setting ObjectContainer.thumb in line 19 fix that more elegantly?

**Comment by ukdtom** [comment]
The code puts this in as a default, and then in some part of the menu system, replace thumb with the one present for a section, a movie, a show or an episode, if present

section_browse

File: Contents/Code/support/items.py [hunk 2]
```diff
@@ -67,6 +70,10 @@ def getItems(key="recently_added", base="library", value=None, flat=False, add_s
         else:
             kind = item.type
 
+        if item.thumb:
```
Comment (pannal) [inline Contents/Code/support/items.py]: Why do you set a global variable here? Isn't setting the thumb in menu_helpers:dig_tree enough?

**Comment by pannal** [comment]
The merge looks weird - didn't you update your master from upstream before pushing the changes? You've got the old camelcase function names as merge source.

File: Contents/Code/support/items.py (hunk 2 above)
Comment (pannal) [inline Contents/Code/support/items.py]: Sorry, you're not setting a global variable here.

**Comment by pannal** [comment]
I didn't want to appear rude, if that's why you closed this pull request :) 

Sorry if I made that impression

**Comment by ukdtom** [comment]
Please check the new PR I made, as well as the PM on the forums :-)

And not rude, but helpful!

/T

**Pull request closed without merging**